apps/site/src/pages/api/link-wordpress.api.ts

```diff
@@ -20,18 +21,28 @@ export default createBaseHandler<ApiSignupRequestBody, ApiSignupResponse>()
     bodyValidator("wordpressInstanceUrl").isURL({
       protocols: ["http", "https"],
       require_protocol: true,
+      require_tld: process.env.NODE_ENV === "production",
       allow_fragments: false,
       allow_query_components: false,
     }),
   )
+  .use(
+    bodyValidator("wordpressSettingsUrl").isURL({
+      protocols: ["http", "https"],
+      require_protocol: true,
+      require_tld: process.env.NODE_ENV === "production",
+      allow_fragments: false,
```

Comment:
I assume we will remove fragments in the WordPress side before submitting to this API. If not I'd allow them and remove them here.

Reply:
Discussed offline. No use case for including fragments (hashes/anchors) in these URLs. They won't be sent by the wordpress plugin as they're never sent to the server anyway